### DISCUSSION TITLE
Dronekit python

### DISCUSSION THREAD

**Discussion by SundaresanN** [HefnySco/OBAL#16] [Ideas]
Hi Hefny, 

This is very interesting project. If we run ardupilot binary as service, is it possible to send command to that through dronekit python library? I believe it's technically possible, just asking if you have tried it.

**Comment by HefnySco** [comment]
Yes sure. 
sudo /home/pi/arducopter -A udp:YOURIP:14550 
make dronekit listen to udp  at YOURIP:port 14550
you can use other ports for example:
sudo /home/pi/arducopter -A udp:YOURIP:14550  -C udp:ANOTHERIP:ANOTHERPORT
